spinn_front_end_common/utility_models/data_speed_up_packet_gatherer_machine_vertex.py

```diff
@@ -626,7 +589,7 @@ def _print_missing(seq_nums):
         last_seq_num = 0
         for seq_num in sorted(seq_nums):
             if seq_num != last_seq_num + 1:
-                print "from list i'm missing sequence num {}".format(seq_num)
+                print "from list i'm missing sequence num {}\n".format(seq_num)
```

Comment:
Not sure you want the \n in a python print (python prints \n anyway)

Reply:
was during a debug as wasnt seeing what i expected. later on found it was somethign very differnet. removed